src/components/tables/AdminReviewsTable.tsx

```diff
@@ -60,6 +109,67 @@ type Props = {
   isError: boolean;
   hasFilters: boolean;
   reviews: AdminReview[] | undefined;
+  totalPages: number | undefined;
+  refetch: Refetch;
```

Comment:
In as much as the type definition for `Refetch` is good, a simple `() => void` would satisfy the same definition for refetch.